packages/curve-ui-kit/src/shared/api/figma-tokens/validation/analyze_figma_tokens.py

```diff
@@ -0,0 +1,36 @@
+import json
+import re
+from collections import defaultdict
+
+def analyze_json_file(file_path):
```

Comment:
what is this?

Reply:
This is a script for analyzing tokens in a file, handy for checking the correctness of hyperlinks within it. I've added some documentation.